### DISCUSSION TITLE
Expo - Infinite loading "Installing Cocoapods..."

### DISCUSSION THREAD

**Discussion by stephanepx** [thebergamo/react-native-fbsdk-next#573] [Q&A]
Hi guys!
Thanks for this library!

I installed this lib on my expo app with this section : https://github.com/thebergamo/react-native-fbsdk-next?tab=readme-ov-file#expo-installation

I added appId, displayName and clientToken.

After that I did : rm -rf .expo ios
And then : npx expo prebuild and npx expo run:ios

My problem is that Installing Cocoapods task is in infinite loop without any logs. 
Do you have any idea how to fix this "error" ? 

here are my modules versions : 
"react-native-fbsdk-next": "^13.1.3",
"expo": "^51.0.2",
 "expo-tracking-transparency": "~4.0.2",
 
 Thanks for your help

**Comment by mikehardy** [comment]
Not without any logs, no, sorry
